docs/advanced-recipes/custom-useatom-hooks.mdx

```diff
@@ -14,12 +14,16 @@ This page shows the ways of creating different utility functions. Utility functi
 import { useAtom } from 'jotai'
 import { selectAtom } from 'jotai/utils'
 
-export function useSelectAtom(anAtom, keyFn) {
-    return useAtom(selectAtom(anAtom, keyFn));
+/* if an atom is created here, please use `useMemo(() => atom(initValue), [initValue])` instead. */

```

Comment:
Hmm, on second thought, changing `useSelectAtom` is not the intention of the original recipe.

So, how about keeping the original function, and adding a usage code snippet?
Something like this:
```js
export function useSelectAtom(anAtom, keyFn) {
    return useAtom(selectAtom(anAtom, keyFn));
}

// how to use it
useSelectAtom(
  useMemo(() => atom(initValue), [initValue]),
  useCallBack((state) => state.prop, [])
)
```

Same for `useFocusAtom`.

Reply:
This one is better